integration-tests/contract-batch-include-reveal.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { OpKind } from '@taquito/taquito';
+import { CONFIGS } from './config';
+
+CONFIGS().forEach(({ lib, rpc, setup, knownBaker }) => {
+    const Tezos = lib;
+    describe(`Test contract.batch using: ${rpc}`, () => {
+        beforeEach(async (done) => {
+            await setup(true);
+            done();
+        });
+        it('Batch estimate including reveal', async (done) => {
```

Comment:
Consider adding a test where `includeRevealOperation` is true but the account do not require a reveal and assert that the fees are the same as if `includeRevealOperation` was false

Reply:
A test case has been added, thank you.